### PR TITLE
Migrated from gradle to maven #1

### PR DESCRIPTION
**Pull request by kshitij1708**

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Handle line endings automatically for files detected as text
```
Comment (mahesh1996) [inline .gitattributes:1]: We should add necessary attributes here as needed. Please remove extra attributes.

**Review by mahesh1996 — CHANGES_REQUESTED** [review]
Please make changes as addressed in review.

File: .gitattributes (hunk 1 above)
Comment (kshitij1708) [reply .gitattributes:1]: These attributes are general purpose attributes and suitable for every web application

**Comment by mahesh1996** [comment]
@kshitij1708 Can you please resolve the conflicts?

**Review by mahesh1996 — APPROVED** [review]

**Pull request merged**